src/DIRAC/WorkloadManagementSystem/Utilities/PilotCStoJSONSynchronizer.py

```diff
@@ -244,6 +244,29 @@ def _getPilotOptionsPerSetup(self, setup, pilotDict):
                     return queueOptionRes
                 queuesDict[queue] = queueOptionRes["Value"]
             pilotDict["Setups"][setup]["Logging"]["Queues"] = queuesDict
+        elif "loggingRESTService" in pilotDict["Setups"][setup]:
+            self.log.debug(
+                "Getting option of ", "/DIRAC/Setups/%s/%s" % (setup, pilotDict["Setups"][setup]["loggingRESTService"])
+            )
+            result = gConfig.getOption(
+                "/DIRAC/Setups/%s/%s" % (setup, pilotDict["Setups"][setup]["loggingRESTService"])
```

Comment:
What you want to use is the Pilot configuration section in the Operations section of the CS.